docs/reference/configuration-examples.md

```diff
@@ -18,7 +18,8 @@ example refers to sensitive values by [using a `secrets.yml` file](secrets-refer
 
 ```yml
 sonarr:
-  - base_url: !secret sonarr_url
+  sonarr:
```

Comment:
As a theme, can we go with something like `instance1`, `series`, `movies`, etc? I think having it say `sonarr` makes it a little disorienting since the section above is *also* named that. Anything different would do. Gives it some contrast and helps make the element a bit more self documenting.

I won't comment on all of them, but all of them should be reviewed and changed accordingly.